G/Geant4/build_tarballs.jl

```diff
@@ -65,15 +84,19 @@ products = [
     LibraryProduct("libG4global", :libG4Global),
     LibraryProduct("libG4tracking", :libG4Tracking),
     LibraryProduct("libG4intercoms", :libG4Intercoms),
+    LibraryProduct("libG4particles", :libG4Particles),
+    LibraryProduct("libG4ptl", :libG4Ptl),
+    FileProduct("share/Geant4/data", :data_dir),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Expat_jll"; compat="2.2.10"),
+    Dependency("Expat_jll"; compat="2.4.8"),
     Dependency("Xorg_libXmu_jll"),
     Dependency("Libglvnd_jll"),
+    Dependency("Xerces_jll"),
     BuildDependency("Xorg_xorgproto_jll"),
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"7", julia_compat="1.6")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"11", julia_compat="1.6")
```

Comment:
Using a too new version of GCC is a compatibility problem, we won't be able to actually use this binary on Julia v1.6.

Reply:
OK. I'll use gcc8, with is the minimal that supports C++17.